docs/src/release_notes/v1.26.md

```diff
@@ -181,13 +100,49 @@ on the release branch in GitHub.
 
 - Added support for collecting `pg_stat_wal` metrics in PostgreSQL 18. (#7005)
 
+- Removed the `ENABLE_AZURE_PVC_UPDATES` configuration, as it is no longer
+  required to resize Azure volumes correctly. The Azure CSI driver includes the
+  necessary fix as of version [1.11.0](https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases/tag/v1.11.0). (#7297)
```

Comment:
```suggestion
  necessary fix as of version [1.11.0](https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases/tag/v1.11.0). (#7297) <-- no 1.25 1.24 1.22 -->
```

Reply:
This feature has been backported to all versions. However, the setting has not been functional since 2023.